clang/tools/dpct/extensions/pytorch_api_rules/pytorch_api.yaml

```diff
@@ -156,7 +156,7 @@
   Kind: Class
   Priority: Takeover
   In: at::Tensor # The underlying type of torch::Tensor is at::Tensor
-  Out: torch::Tensor
+  Out: null
```

Comment:
null is a new keyword here, how about if Out field is not given, then it is treat as ignored?